### PR TITLE
ttyplot 1.4 (new formula)

### PR DESCRIPTION
**Pull request by Amar1729**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Neat command-line realtime plotting utility:  
https://github.com/tenox7/ttyplot

### CODE REVIEW

File: Formula/ttyplot.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  desc "Realtime plotting utility for terminal with data input from stdin"
+  homepage "https://github.com/tenox7/ttyplot"
+  url "https://github.com/tenox7/ttyplot/archive/1.4.tar.gz"
+  sha256 "11974754981406d19cfa16865b59770faaf3ade8d909d9a0134dc56e00d29bd4"
+
+  uses_from_macos "ncurses"
+
+  def install
+    system "make"
+    bin.install "ttyplot"
```
Comment (SMillerDev) [inline Formula/ttyplot.rb]: Is there no `make install`?

Comment (Amar1729) [reply Formula/ttyplot.rb]: In 1.4, no :/  
there is a `make install` in current HEAD though.

**Comment by SMillerDev** [comment]
Thanks @Amar1729! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by SMillerDev** [comment]
@BrewTestBot test this please

**Pull request closed without merging**